mender-deb-package

```diff
@@ -80,10 +80,15 @@ get_deb_version() {
     DEB_VERSION="$(echo ${DEB_VERSION} | sed -E 's/([0-9]+\.)([0-9]+)\.[0-9]+/echo "\1$((\2+1))\.0"/e')"
     # Append git date and commit
     DEB_VERSION="${DEB_VERSION}$(git log -1 --pretty=~git%cd.%h --date format:%Y%m%d)"
+    DEB_VERSION_NO_REV="${DEB_VERSION}"
```

Comment:
`DEB_VERSION_NO_REV` needs to be defined also on tags, line 73 above

Reply:
ah right! I thought this line was covered with tests. thank you.

Reply:
There are no tests for this script